app/controllers/api/encryptables_controller.rb

```diff
@@ -121,4 +124,12 @@ def permitted_attrs
       permitted_attrs
     end
   end
+
+  def log_read_access
+    encryptable.paper_trail.save_with_version
+    v = encryptable.versions.first
+    v.event = 'viewed'
```

Comment:
```suggestion
    v.event = :viewed
```

possible to use symbol here?